test/dotnet-publish3.Tests/GivenDotnetPublish3PublishesProjects.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Publish3.Tests
+{
+    public class GivenDotnetPublish3PublishesProjects : TestBase
+    {
+        [Fact]
+        public void ItPublishesARunnablePortableApp()
```

Comment:
Can't do it yet since we don't have restore3 for a specific RID.  Waiting until that comes online.